core/src/main/java/com/hcl/ecomm/core/services/impl/UserComplaintFormWorkflowProcess.java

```diff
@@ -84,37 +95,53 @@ public void execute(WorkItem workItem, WorkflowSession workflowSession, MetaData
 				persistComplaintIndb(xmlDocument, "Closed");
 			}
 
-	 	    } catch (PathNotFoundException e1) {
-		 	     LOG.error("PathNotFoundException while getting the Data.xml node. Error={}",e1);
-		 	    } catch (RepositoryException e1) {
-		 	    	 LOG.error("RepositoryException while getting the Data.xml node. Error={}",e1);
-		 	    } catch (Exception e) {
-		 	    	LOG.error("Error while update jcr:data. Error={}" + e);
-	 	    } 
-	 	   LOG.debug("UserComplaintFormWorkflowProcess execute end");
+	 	    } catch (PathNotFoundException e) {
```

Comment:
@rakeshkumar-verma : can we not use a generic catch exception rather tan multiple catch block here ?

Reply:
@rishabh-behl  added multiple catch intentionally as in system getting error in this section.